app/javascript/src/Policies/actions/ThunkActions.jsx

```diff
@@ -0,0 +1,122 @@
+// @flow
+
+import { uiComponentTransition } from 'Policies/actions/UISettings'
+import { fetchRegistry, loadRegistrySuccess } from 'Policies/actions/PolicyRegistry'
+
+import {
+  addPolicyToChain,
+  removePolicy,
+  updatePolicyInChain,
+  fetchChain,
+  loadChain
+} from 'Policies/actions/PolicyChain'
+
+import { updatePolicyConfig } from 'Policies/actions'
+
+import type { UIComponent } from 'Policies/actions/UISettings'
```

Comment:
Why not importing in same line as `import { uiComponentTransition } from 'Policies/actions/UISettings'`

Reply:
This one is a type

Reply:
You are right